broker/server.js

```diff
@@ -131,10 +130,42 @@ app.post("/Observation", (req, res) => {
     });
 });
 
+// Tell Mirth to send an HL7 message
+app.post("/SendHL7Message", (req, res) => {
```

Comment:
Similar comment I had for Antony's latest PR: To avoid making server.js a giant monolithic file that is impossible to support, please move this method in a new directory/file named broker/endpoints/SendHL7Message.js. You will have to export the method in that file.

Then replace the code in server.js with:

const SendHL7MessageEndpoint = require("./endpoints/SendHL7Message.js");
app.get("/SendHL7Message*", SendHL7Message);

Reply:
Done! How does it look now?